api.js

```diff
@@ -0,0 +1,136 @@
+const fetch = require('node-fetch');
+
+const WebhookService = require('../services/webhook-service');
+const TranslationService = require('../services/translation-service');
+const { onshapeApiUrl } = require('../utils');
+const redisClient = require('../redis-client');
+    
+const apiRouter = require('express').Router();
+
+/**
+ * Get the Elements of the current document/workspace.
+ * 
+ * GET /api/elements
+ *      -> 200, [ ...elements ]
+ *      -or-
+ *      -> 500, { error: '...' }
+ */
+apiRouter.get('/elements', (req, res) => {
+    fetch(`${onshapeApiUrl}/documents/d/${req.query.documentId}/w/${req.query.workspaceId}/elements`, {
+        headers: { 'Authorization': `Bearer ${req.user.accessToken}` }
+    })
+        .then((resp) => resp.text())
+        .then((data) => res.status(200).send(data))
```

Comment:
Instead of duplicating, maybe consider making instances of this pattern (fetch from OS and return) into a function? something like `forwardRequestToOnshape(onshapePath, res, req)`

Reply:
Updated.